ExtraDry/Sample.Client/ViewModels/Components/StandardComponentsMenu.cs

```diff
@@ -10,6 +10,9 @@ public class StandardComponentsMenu {
     [Navigation(Icon = "tri-check")]
     public string TriCheck => $"/components/standard/tri-check";
 
+    [Navigation]
```

Comment:
Can you add an icon please?  I've found one from Font Awesome and have downloaded and posted in Teams.

Reply:
Updated with the icon.